pkg/tsdb/parca/resources.go

```diff
@@ -56,74 +59,94 @@ func (d *ParcaDatasource) callProfileTypes(ctx context.Context, req *backend.Cal
 
 	data, err := json.Marshal(types)
 	if err != nil {
+		logger.Debug("callProfileTypes errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
 	err = sender.Send(&backend.CallResourceResponse{Body: data, Headers: req.Headers, Status: 200})
 	if err != nil {
+		logger.Debug("callProfileTypes errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
+
+	logger.Debug("callProfileTypes succeeded")
 	return nil
 }
 
 func (d *ParcaDatasource) callLabelNames(ctx context.Context, req *backend.CallResourceRequest, sender backend.CallResourceResponseSender) error {
+	logger.Debug("callLabelNames called")
+
 	ctx, span := tracing.DefaultTracer().Start(ctx, "datasource.parca.callLabelNames")
 	defer span.End()
 	res, err := d.client.Labels(ctx, connect.NewRequest(&v1alpha1.LabelsRequest{}))
 	if err != nil {
+		logger.Debug("callLabelNames errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
 
 	data, err := json.Marshal(res.Msg.LabelNames)
 	if err != nil {
+		logger.Debug("callLabelNames errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
 	err = sender.Send(&backend.CallResourceResponse{Body: data, Headers: req.Headers, Status: 200})
 	if err != nil {
+		logger.Debug("callLabelNames errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
+
+	logger.Debug("callLabelNames succeeded")
 	return nil
 }
 
 func (d *ParcaDatasource) callLabelValues(ctx context.Context, req *backend.CallResourceRequest, sender backend.CallResourceResponseSender) error {
+	logger.Debug("callLabelValues called")
+
 	ctx, span := tracing.DefaultTracer().Start(ctx, "datasource.parca.callLabelValues")
 	defer span.End()
 	parsedUrl, err := url.Parse(req.URL)
 	if err != nil {
+		logger.Debug("callLabelValues errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
 	label, ok := parsedUrl.Query()["label"]
 	if !ok {
+		logger.Debug("call to parsedUrl.Query not ok")
 		label = []string{""}
 	}
 	res, err := d.client.Values(ctx, connect.NewRequest(&v1alpha1.ValuesRequest{LabelName: label[0]}))
 	if err != nil {
+		logger.Debug("callLabelValues errored", "error", err)
 		span.RecordError(err)
 		span.SetStatus(codes.Error, err.Error())
 		return err
 	}
 	data, err := json.Marshal(res.Msg.LabelValues)
 	if err != nil {
+		logger.Debug("callLabelValues errored", "error", err)
```

Comment:
Please mention here that it errored in the marshal (and below that it errored in the send).